src/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/EventTask.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+**
+** Purpose:
+** This public class describes the metadata for a specific Task
+** defined by a Provider. An instance of this class is obtained
+** from a ProviderMetadata object.
+**
+============================================================*/
```

Comment:
@danmosemsft  should we remove this descripition from all the classes ? we dont have it anywhere else.